File: level 41/classwork/classwork.py
# საკლასო დავალება:

# შექმენით dictionary, სადაც იქნება შემდეგი გასაღებები: name, surname, academy, fav-color, city, goa-student, age, fav-programming-lans
stats = {
    "name": "soso",
    "surname": "rtveliashvili",
    "academy": "GOA",
    "fav-color": "green",
    "goa student": True,
    "fav-programming-lang":"python"
}
print(stats["name"])
print(stats["surname"])
print(stats["academy"])
print(stats["fav-color"])
print(stats["goa student"])
print(stats["fav-programming-lang"])


for i in stats.keys():
    print(i)

for i in stats.values():
    print(i)

for i,s in stats.items():
    print(i,s)

for r,u in stats.items():
    print(f"{r} {u}")

File: level 67/classwork/classwork.js
let myself = {
    name: "soso",
    surname: "rtveliashvili",
    academy: "Goa",
    city: "Tbilisi",
    role: "mini-leader",
    favcolor: "green",
    
    fullname() {
        console.log(this.name +" "+ surname)
    },
    color() {
        console.log(this.favcolor)
    }

}
myself.city="kaspi"
console.log(myself.city)
console.log(myself)
console.log(myself.academy)
myself.fullname()

function checking() {
    let check = document.Basic.check.checked;
    let check2 = document.Basic.anotheCheck.checked;
    console.log(check && check2);
    console.log(check || check2);
}
checking()